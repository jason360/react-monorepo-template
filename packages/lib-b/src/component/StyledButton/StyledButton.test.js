import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import { StyledEngineProvider } from '@mui/material/styles';
import StyledButton from './StyledButton';

describe('<StyledButton />', () => {
    test('renders StyledButton correctly', () => {
        render(
            <StyledEngineProvider injectFirst>
                <StyledButton color="primary">Primary</StyledButton>
            </StyledEngineProvider>
        );
        expect(screen.getByRole('button')).toMatchSnapshot();
    });
});