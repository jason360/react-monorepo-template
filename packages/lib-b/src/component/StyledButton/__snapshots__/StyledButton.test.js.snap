// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledButton /> renders StyledButton correctly 1`] = `
<button
  class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root css-d1czvq-MuiButtonBase-root-MuiButton-root"
  tabindex="0"
  type="button"
>
  Primary
  <span
    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
  />
</button>
`;
